src/main/kotlin/com/matthewrussell/trchunkbrowser/app/mainview/MainView.kt

```diff
@@ -25,9 +26,31 @@ class MainView : View() {
     override val root = stackpane {
         vbox {
             addClass(MainViewStyles.root)
-            add(JFXSnackbar(this).apply {
+            add(JFXSnackbar(this).apply { // Progress snackbar
+                viewModel.snackBarProgress.subscribe { progress ->
+                    if(progress < 100) {
+                        enqueue(JFXSnackbar.SnackbarEvent(
+                            messages["export_in_progress"],
+                            "",
+                            0,
+                            true,
+                            null))
```

Comment:
when you put parameters on their own lines, the close parentheses go on their own lines as well